config/test_settings.py

```diff
@@ -0,0 +1,13 @@
+from decouple import config
+import dj_database_url
+
+from config.settings import *  # noqa F403
```

Comment:
not really nice, maybe better way to do this?

Reply:
taken from pybites books tests: https://github.com/PyBites-Open-Source/pybitesbooks/blob/develop/myreadinglist/test_settings.py